rootfs/scheduler/mock.py

```diff
@@ -391,7 +391,7 @@ def upsert_pods(controller, url):
     # turn RC / RS (which a Deployment creates) url into pods one
     url = url.replace(cache_key(controller['metadata']['name']), '')
     if '_replicasets_' in url:
-        url = url.replace('_replicasets_', '_pods').replace('apis_extensions_v1beta1', 'api_v1')  # noqa
+        url = url.replace('_replicasets_', '_pods').replace('apis_apps_v1', 'api_v1')  # noqa
```

Comment:
Shouldn't this replace both the old and the new string?

Reply:
Yes, I was thinking about this yesterday. I think this is okay this way because it is just for the mocking of the api routes but you are right. We should be replacing both the old and the new api if this was in the actual scheduler code.

Reply:
This was addressed in d8e65a9. Decided it was best to just replace both if each one exist. For the mock.py it doesn't really matter, but it just looks cleaner and someone else looking at the code now has a comment to refer to and understand why this is being done. :+1: